android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/ui/component/ActorCard.kt

```diff
@@ -0,0 +1,89 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.ui.theme.Shapes
+import agency.five.codebase.android.movieapp.ui.theme.Typography
+import agency.five.codebase.android.movieapp.ui.theme.spacing
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import coil.compose.AsyncImage
+
+
+data class ActorCardViewState(
+    val imageUrl: String,
+    val name: String,
+    val character: String,
+)
+
+@Composable
+fun ActorCard(
+    modifier: Modifier = Modifier,
+    actorCardViewState: ActorCardViewState
+) {
+    Card(
+        elevation = dimensionResource(id = R.dimen.card_elevation),
+        modifier = modifier,
+        shape = Shapes.large
+    )
+    {
+        Column {
+            AsyncImage(
+                model = actorCardViewState.imageUrl,
+                contentDescription = actorCardViewState.name,
+                modifier = Modifier
+                    .height(dimensionResource(id = R.dimen.actor_image_height)),
```

Comment:
Because you are defining size of card from outside, this won't look well if we pass very large or very small height (ex. 500.dp, 100.dp)
Try using `Modifier.weight` extension function instead.